lua/lspconfig/util.lua

```diff
@@ -401,4 +393,9 @@ function M.find_mercurial_ancestor(startpath)
   return vim.fs.find('.hg', { path = startpath, upward = true })
 end
 
+--- @deprecated use `vim.fs.find('node_modules', { path = startpath, upward = true })` instead
+function M.find_node_modules_ancestor(startpath)
+  return vim.fs.find('node_modules', { path = startpath, upward = true })
```

Comment:
vim.fs.find return a table before we return string.

Reply:
Oh shit, you're right. Thanks.

Reply:
I guess I need to wrap all `vim.fs.find` with `next()`?